packages/cryptobox/src/main/store/CryptoboxCRUDStore.ts

```diff
@@ -154,6 +153,20 @@ class CryptoboxCRUDStore implements ProteusSession.PreKeyStore {
       });
   }
 
+  public read_sessions(identity: ProteusKeys.IdentityKeyPair): Promise<Array<ProteusSession.Session>> {
+    return this.engine.readAll(CryptoboxCRUDStore.STORES.SESSIONS).then((records: Array<any>) => {
```

Comment:
```ts
return this.engine.readAll(CryptoboxCRUDStore.STORES.SESSIONS).then((records: PersistedRecord[]) => {
```

Reply:
We tried to this in the past but we received:

> src/main/store/CryptoboxCRUDStore.ts:157:73 - error TS2345: Argument of type '(records: PersistedRecord[]) => Session[]' is not assignable to parameter of type '((
value: {}[]) => Session[] | PromiseLike<Session[]>) | null | undefined'.
  Type '(records: PersistedRecord[]) => Session[]' is not assignable to type '(value: {}[]) => Session[] | PromiseLike<Session[]>'.
    Types of parameters 'records' and 'value' are incompatible.
      Type '{}[]' is not assignable to type 'PersistedRecord[]'.
157     return this.engine.readAll(CryptoboxCRUDStore.STORES.SESSIONS).then((records: PersistedRecord[]) => {
                                                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~